../rpmio/rpmlua.h:5:10: fatal error: lauxlib.h: No such file or directory

```
depbase=`echo parsePreamble.lo | sed 's|[^/]*$|.deps/&|;s|\.lo$||'`;\
/bin/bash ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H   -I.. -I.. -I../include/ -I../misc  -D_REENTRANT -Wall -Wpointer-arith -Wmissing-prototypes -Wstrict-prototypes  -fno-strict-aliasing -fstack-protector -Wempty-body -fopenmp -g -O2 -MT parsePreamble.lo -MD -MP -MF $depbase.Tpo -c -o parsePreamble.lo parsePreamble.c &&\
mv -f $depbase.Tpo $depbase.Plo
libtool: compile:  gcc -DHAVE_CONFIG_H -I.. -I.. -I../include/ -I../misc -D_REENTRANT -Wall -Wpointer-arith -Wmissing-prototypes -Wstrict-prototypes -fno-strict-aliasing -fstack-protector -Wempty-body -fopenmp -g -O2 -MT parsePreamble.lo -MD -MP -MF .deps/parsePreamble.Tpo -c parsePreamble.c  -fPIC -DPIC -o .libs/parsePreamble.o
In file included from parsePreamble.c:15:
../rpmio/rpmlua.h:5:10: fatal error: lauxlib.h: No such file or directory
```

The distro is Ubuntu eoan, liblua5.3-dev is installed, `configure.ac` was modified to find it using pkg-config. The configure flags are: `--with-lua`.

--- a/lib/rpmliblua.c
+++ b/lib/rpmliblua.c
@@ -1,6 +1,7 @@
 #include "system.h"
 
 #include <lua.h>
+#include <lauxlib.h>
 
 #include <rpm/rpmlib.h>
 

--- a/rpmio/rpmlua.c
+++ b/rpmio/rpmlua.c
@@ -179,8 +179,9 @@ rpmlua rpmluaFree(rpmlua lua)
     return NULL;
 }
 
-void rpmluaRegister(rpmlua lua, const luaL_Reg *funcs, const char *lib)
+void rpmluaRegister(rpmlua lua, const void *regfuncs, const char *lib)
 {
+    const luaL_Reg *funcs = regfuncs;
     lua_getfield(lua->L, LUA_REGISTRYINDEX, LUA_LOADED_TABLE);
     lua_getfield(lua->L, -1, lib);
     luaL_setfuncs(lua->L, funcs, 0);

--- a/rpmio/rpmlua.h
+++ b/rpmio/rpmlua.h
@@ -1,10 +1,6 @@
 #ifndef RPMLUA_H
 #define RPMLUA_H
 
-#ifdef WITH_LUA
-#include <lauxlib.h>
-#endif
-
 typedef enum rpmluavType_e {
     RPMLUAV_NIL		= 0,
     RPMLUAV_STRING	= 1,
@@ -23,9 +19,7 @@ rpmlua rpmluaNew(void);
 rpmlua rpmluaFree(rpmlua lua);
 rpmlua rpmluaGetGlobalState(void);
 
-#ifdef WITH_LUA
-void rpmluaRegister(rpmlua lua, const luaL_Reg *funcs, const char *lib);
-#endif
+void rpmluaRegister(rpmlua lua, const void *regfuncs, const char *lib);
 
 int rpmluaCheckScript(rpmlua lua, const char *script,
 		      const char *name);

